src/chronometer.js

```diff
@@ -1,33 +1,45 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime ++
+      if (typeof printTimeCallback === 'function') {
+        printTimeCallback()
+      }
+    }, 1000)
   }
 
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime / 60)
   }
 
   getSeconds() {
-    // ... your code goes here
+    return Math.floor(this.currentTime % 60)
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    if (value < 10) {
+      return '0' + value;
+    }
+    return '' + value;
   }
 
```

Comment:
Aqui está quase lá, seu raciocínio está indo pelo melhor caminho. Mas, precisa transformar tudo em string, para poder retornar conforme o lab pede. Teremos 3 tipos de retorno então. Teremos '00', 'string', '0 + valor em string'. 
Vamos transformar primeiro tudo em string: let valueString = value.toString( )
Agora vamos criar as condições do if, são três condições, para termos os 3 retornos: 
se o valor for igual, identico a valor e tipo, 0 (zero); retorna '00' /
se o valor for igual ou maior de 10; retorna o valor transformado em string -> valueString /
senão, ai retorna '0' + valueString 
Tenta fazer e me fala :)